### ISSUE TITLE
Undefined name '__builtin__'

### ISSUE THREAD

**Issue by OverLordGoldDragon** [spyder-ide/spyder#12585]
`__builtins__` isn't flagged, but `__builtin__` is also a default in the global namespace. Is this Spyder or should I head to Pyflakes?

![image](https://user-images.githubusercontent.com/16495490/80908376-717a2b80-8d30-11ea-822d-3f051bc44423.png)

**Comment by OverLordGoldDragon** [comment]
Well, somehow this is always imported in `__main__`, but does raise exception in non-main modules; `import builtins` is the fix. Still, at least in `__main__`, this is a false positive.

**Comment by goanpeca** [comment]
@OverLordGoldDragon could you check with pyflakes outside of Spyder to check:

> Is this Spyder or should I head to Pyflakes?

?

Thanks!

**Comment by OverLordGoldDragon** [comment]
TL;DR `__builtin__` is defined "interactively" (IPython?), not in `__main__`, which can be confirmed by "running" `__main__` instead of its snippets.
